### ISSUE TITLE
file attributes auto update

### ISSUE THREAD

**Issue by only1chunts** [gigascience/gigadb-website#34]
The development company were asked to include some code that would auto update certain file attributes whenever a file was added to a dataset. They tried, but obviously it doesn't work properly! This code needs removing. Eventually I would like it replaced with some that actually works. For an example of the sort of thing that gets added see  http://gigadb.org/dataset/100198 the file attributes of the files preliminary_platanus_assembly_pig_* have all been added pointlessly by the script when I added the files via the admin pages (NB this doesn't happen when files are added directly to the database via Jesse's scripts).

**Comment by only1chunts** [comment]
in jesse's branch /fix/bugs/ I cannot test this function as I get an error 500 when I try to add a file.
![image](https://cloud.githubusercontent.com/assets/6037145/15766507/e09726b6-2971-11e6-979f-c9d77853e7fd.png)
It appears you have now made the addition of a Sample mandatory?!  as it worked when I added a random sample.
On the plus side, when I did add the sample name the attributes problem does appear to be fixed.
